fec_integration_tests/file_convertor_tests/test_convert_to_json_machine.py

```diff
@@ -0,0 +1,79 @@
+import filecmp
+import os
```

Comment:
There are two physical test boards spinn-4 which does not sit behind a spalloc server and spinn-2 which sits behind the spalloc server on port 22245.

So we can not test against spinn_5 as there is no spin 5

the second test for "spinn_4" is one that tests a weird machine.  No need to repeat that for just because the machine came via spalloc.

Reply:
ok. i thought it was a made up board. but that makes more sense. ok.